modules/web/src/app/cluster-template/component.ts

```diff
@@ -139,6 +141,7 @@ export class ClusterTemplateComponent implements OnInit, OnChanges, OnDestroy {
         this.dataSource.data = this.templates;
         this.isInitializing = false;
         this._loadDatacenters();
+        this.clusterTemplateFragment && this._onCreateClusterFragment();
```

Comment:
After using router `state` for navigation in cluster wizard.
```suggestion
        const createClusterFromTemplateID = history?.state?.data?.createClusterFromTemplateID;
        // remove _onCreateClusterFragment method and this. clusterTemplateFragment property
        if (createClusterFromTemplateID) {
          const clusterTemplate = this.templates.find((ct: ClusterTemplate) => ct.id === createClusterFromTemplateID);
          if (clusterTemplate) {
            this.createCluster(clusterTemplate);
          }
        }
```